metaci/release/utils.py

```diff
@@ -220,14 +222,20 @@ def send_stop_webhook(release, role, config_item):
     }
     token = jwt_for_webhook()
     response = requests.post(
-        f"{settings.METACI_RELEASE_WEBHOOK_URL}/implementation_step_id/{implementation_step_id}/stop/",
+        f"{settings.METACI_RELEASE_WEBHOOK_URL}/implementation/{implementation_step_id}/stop/",
         json=payload,
         headers={"Authorization": f"Bearer {token}"},
     )
     result = response.json()
-    if result["success"]:
+    if (
+        "results" in result
+        and len(result["results"]) > 0
+        and result["results"][0]["success"]
+        or result["success"]
```

Comment:
The example success response shown at https://confluence.internal.salesforce.com/display/CHANGEMGMT/Change+APIs#expand-Start does not include a `success` key at the top level -- so I would expect this to raise KeyError